### DISCUSSION TITLE
Are nodes built-in now?

### DISCUSSION THREAD

**Discussion by TheCoolestName** [bnpr/Malt#107] [Q&A] [answered]
and if so, how/where do I download it?
Actually, I just found out 😅 sorry I was being an idiot. but now, how do I use textures?

**Comment by pragma37** [comment, marked as answer]
Nodes are only available in the Development branch for now.

Here's a textured material example:
![imagen](https://user-images.githubusercontent.com/36610999/127776347-d491627d-8588-4291-8075-6e816bba3526.png)

You can find an example file in #94.
